src/simcore_service_deployment_agent/models.py

```diff
@@ -9,3 +12,20 @@ class ComposeSpecsDict(TypedDict):
     version: str
     services: dict[ServiceName, ServiceSpecsDict]
     volumes: dict[VolumeName, Any]
+
+
+class WebserverExtraEnvirons(BaseModel):
```

Comment:
Ok here I have a question.
Do we want to make the deployer-agent webserver dependent? I would say no if possible.
Would it not be possible to configure the deployer to pass whatever additional ENVs instead?